networking/networking.go

```diff
@@ -229,18 +233,41 @@ func Load(podRoot string, podID *types.UUID) (*Networking, error) {
 	}, nil
 }
 
-func (n *Networking) GetDefaultIP() net.IP {
-	if len(n.nets) == 0 {
-		return nil
+// GetDefaultNet iterates through all loaded networks and returns either
+// the first network that has masquerading enabled,
+// or the last network in case there is no masqueraded one,
+// or an error if no network was loaded.
```

Comment:
> Alternatively, since this special network ordering is only for port forwarding, shouldn't they be moved to networking/portfwd.go?

This is not the whole truth, it's directly related to CNI's IP masquerading rules which also influences rkt's port forwarding. Let me see if I can clarify the name and location of these functions.